### DISCUSSION TITLE
[Proposition] Ajout d'une fonctionnalité "Bonnes pratiques complémentaires"

### DISCUSSION THREAD

**Discussion by florinesueur** [cnumr/best-practices-wordpress#102] [Site]
La réflexion est venue car je trouve qu' il y a souvent une référence à une autre BP en tant que complémentaire et que celle-ci se situe dans le même scope.
Aussi dès qu'il y a un changement de nom de fiche ou pire peut-être un changement de numérotation, il faudra revenir manuelle sur chaque fiche changer le lien.
Du coup, je propose le développement d'un petit compenent qui ira chercher une liste des bp du même scope que la bp en cours.

Qu'en pensez vous ?

**Comment by hrenaud** [comment]
Je n'avais pas vu ça, on peut le noter pour la v2 du site.

**Comment by hrenaud** [comment]
Je pense qu'une fois la bp/lexique est validée, il ne faut plus changer sur nom et slug, car sinon, on risque de casser les liens vers-celle-ci.

Mettre cette règle en place n'empêche pas de réfléchir à cette fonctionnalité.
On peut ajouter dans les meta une liste de slug complémentaire et les afficher quelque part dans la page
